stencil_main.py

```diff
@@ -24,57 +21,16 @@
     validate_outfield,
 )
 from functions import evaluate
-    #evaluate.add_data
-    #evaluate.runtimedevelopment
-    
+
 from functions import stencils_numpy
-    #(
-    #stencils_numpy.test,
-    #stencils_numpy.laplacian1d,
-    #stencils_numpy.laplacian2d,
-    #stencils_numpy.laplacian3d,
-    #stencils_numpy.FMA,
-    #stencils_numpy.lapoflap1d,
-    #stencils_numpy.lapoflap2d,
-    #stencils_numpy.lapoflap3d,
-#)
-from functions import stencils_numbajit
-#(
-#    test_numba,
-#    laplacian_numba,
-#    FMA_numba,    
-#) 
 from functions import stencils_numbaloop
-#(
-#    laplacian1d,    
-#    laplacian2d,
-#    laplacian3d,
-#) 
-from functions import stencils_numbastencil 
-#(
-#    laplacian1d,
-#    laplacian2d,
-#    laplacian3d,
-#    laplacian1d_numbastencil_help,
-#    laplacian2d_numbastencil_help,
-#    laplacian3d_numbastencil_help,
-#)
-from functions import stencils_numbavectorize
-#(
-#    FMA,
-#)  # , laplacian1d_numbavectorize
+from functions import stencils_numbastencil
+from functions import stencils_gt4py

```

Comment:
Provided that `functions/gt4py_numpy.py` is renamed as `functions/stencils_gt4py.py`.